### PR TITLE
Queries

### PR DESCRIPTION
**Pull request by Iqbal-Elham**
## Vet clinic database: create animals table
This is the First project of module 4, and I create a database with an animals table.
### The requirements needed:
- Using Gitflow.
- Documented in a professional way.
- Create a database named `vet_clinic`.
- Create a table named `animals`
- Take a screenshot of the results of the queries.
first
second

third

### CODE REVIEW

**Review by fobadara — COMMENTED** [review]
Hi @Iqbal-Elham, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a live demo section without a live demo link. You can remove the live demo section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: data.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+
+INSERT INTO animals (name, date_of_birth, escape_attempt, neutered, weight_kg) VALUES ('Aguman', '2020-02-03', 0, true, 10.23);
```
Comment (Gambit142) [inline data.sql]: - [ ] The name of the animal is `Agumon` and not `Aguman`. The implication of inserting incorrect data into the database is that queries will not return the correct results like the first query in your `queries.sql` file. Kindly insert the correct name. 💻 😎 

When you make the change, kindly update the pull request description with the screenshots of the correct result. 👍

File: queries.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+select * from animals where name like '%mon%';
+select name from animals where date_of_birth between '2016-01-01' and '2019-12-30';
+select name from animals where neutered = true and escape_attempt < 3;
+select date_of_birth from animals where name in ('Agumon', 'Pikachu');
```
Comment (Gambit142) [inline queries.sql:5]: - [ ] This query returns the wrong result. Kindly check comment https://github.com/Iqbal-Elham/Vet-Clinic-Database/pull/1#discussion_r1157558231 for the appropriate explanation. 👍 😎

**Review by Gambit142 — APPROVED** [review]
Hi @Iqbal-Elham 👋, 

Good job so far!

## Highlights
- Everything that creates or changes the table's structure is saved in the schema.sql file ✔️ 
- Everything that populates tables with data is saved in the data.sql file ✔️ 
- Everything that is a query that helps to analyze data is saved in the queries.sql file ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Iqbal-Elham, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**